app/controllers/usersController.js

```diff
@@ -29,4 +33,77 @@ const addUser = (req, res) => {
     });
 };
 
-module.exports = { addUser };
+const signIn = (req, res) => {
+  const { email, password } = req.body;
+
+  if (email === undefined) {
+    logger.info('No input email!');
+    return res.status(400).send('No input email!');
+  }
+
+  if (password === undefined) {
+    logger.info('No input password!');
+    return res.status(400).send('No input password!');
+  }
+
+  return obtainAllUsers({ where: { email } }).then(result => {
+    if (!isEmailValid(email)) {
+      logger.info(`The email: ${email} is not a valid WOLOX email.`);
+      return res.status(200).send(`The email: ${email} is not a valid WOLOX email.`);
+    }
+
+    if (result[0] === undefined) {
+      logger.info(`The email: ${email} is not registered.`);
+      return res.status(500).send(`The email: ${email} is not registered.`);
+    }
+
+    if (!checkPassword(password, result[0].password)) {
+      logger.info(`The password for the user with the email: ${email} was wrong.`);
+      return res.status(200).send(`The password for the user with the email: ${email} was wrong.`);
+    }
+
+    const token = signToken(email);
+
+    logger.info('Password OK, token sended.');
+    return res.status(200).send({ auth: true, token });
+  });
+};
+
+const listUsers = (req, res) => {
+  const token =
+    (req.body && req.body.access_token) ||
+    (req.query && req.query.access_token) ||
+    req.headers['x-access-token'];
+
+  const { page } = req.query;
+  const { limit } = req.query;
+
+  if (token === undefined) {
+    logger.info('The token was not given');
+    return res.status(500).send('The token was not given');
+  }
+
+  return obtainAllUsers({ where: { email: verifyToken(token).email } }).then(result => {
```

Comment:
This validstion should not happen here.
This should go to a middleware.
Also it would be nicer to use in this case a findOne method, because you just want one.